### ISSUE TITLE
Intersection doesn't allow bicycles to pass

### ISSUE THREAD

**Issue by xivk** [anyways-open/cycling-frontend#59]
https://routeplanner.bike.brussels/?sb=true&o=4.3317550,50.8767605&d=4.3979127,50.8699507&p=profile1#18.03/50.85747/4.30949?sb=true&o=4.3081248,50.8573845&d=4.3103418,50.8572934&p=profile1

![image](https://user-images.githubusercontent.com/1128068/84947217-bf30d480-b0e1-11ea-9c71-23d67ace27e6.png)

It's unclear how to fix this, I created a note here:

https://www.openstreetmap.org/note/2233086

**Comment by pietervdvn** [comment]
I too had a look at this; this is one of those typical clusterfuck crossroads.

I doublechecked the profile, it is allowed to take the sidewalks (under a 'dismount'-condition) but it is ~ten times slower then normal cycling under the Brussels profile, so the detour is preferred here. (AKA I'm not gonna change the routing profiles just for this case).

However, I spotted a small 'entrance' for cyclists and a small exit, hinting that a cyclist can go straight onto the middle part and continue to the other street to cross (Mapilarry helped with that) so I'm gonna fix it.
